apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx

```diff
@@ -41,21 +42,31 @@ const OrganizationEnterpriseIdentityProviderCreateWizardHelp:
             ["data-testid"]: testId
         } = props;
 
+        const { t } = useTranslation();
+
         return (
             <>
                 <div data-testid={ testId }>
-                    <Heading as="h5">Name</Heading>
-                    <p>Provide a unique name for the enterprise authentication provider so that it can be
-                    easily identified.</p>
+                    <Heading as="h5">{
+                        t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                            ".name.heading")
+                    }</Heading>
+                    <p>{ t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                        ".name.description") }</p>
                     <p>E.g., MyOrgEnterpriseAuthProvider.</p>
                 </div>
 
                 <Divider/>
 
                 <div data-testid={ testId }>
-                    <Heading as="h5">Description</Heading>
-                    <p>Provide a description for the enterprise authentication provider to explain more about it.</p>
-                    <p>E.g., This is the authenticator for MyOrg, which acts as the IDP for MyApp.</p>
+                    <Heading as="h5">{
+                        t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                            ".description.heading")
+                    }</Heading>
+                    <p>{ t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                        ".description.description") }</p>
```

Comment:
```suggestion
                    <p>
                        { t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
                        ".description.description") }
                    </p>
```

Reply:
Check the other places as well.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/0d2293a9f5ef88031f68e9c05ffb4847f410fef8